app/containers/MessageBox/ReplyPreview.js

```diff
@@ -63,21 +76,54 @@ export default class ReplyPreview extends Component {
 	close = () => {
 		const { close } = this.props;
 		close();
-	}
+	};
+
+	renderImage = () => {
+		const {
+			message, baseUrl, user
+		} = this.props;
+
+		if (!message.attachments) {
+			return null;
+		} else {
+			const p = [];
```

Comment:
Rename this variable... this is not recommended.

Reply:
Changed the name of variable